modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -119,6 +118,13 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+  $login-container: 330px;
```

Comment:
container widths inconsistent with design, and needed to properly address new logo